wikimedia/wikipedia-ios
# T376353

**Phabricator:** 
https://phabricator.wikimedia.org/T376353
https://phabricator.wikimedia.org/T376355

### Notes
* Added 5px of padding to align the info icon as well

### Test Steps
1. In an account with edits - follow the steps to get year in review up
2. Check the fourth slide and see if it shows the number of edits 
3. Repeat steps with account that does not

### Screenshots/Videos
